### ISSUE TITLE
Cannot `bazel run` shell scripts on Windows

### ISSUE THREAD

**Issue by alexeagle** [bazelbuild/rules_nodejs#9]
`bazel run @yarn//:yarn` fails on windows with
`C:/path/to/output-base/external/yarn/yarn.sh is not recognized as an internal or external command, operable program or batch file`

The workaround is
```sh
bazel build @yarn//:yarn
sh -c $(bazel info output_base)/external/yarn/yarn.sh
```

/cc @dslomov

**Comment by davidstanke** [comment]
Should this be raised in bazelbuild/bazel? It doesn't seem specific to node.

**Comment by alexeagle** [comment]
@davidstanke probably, we'll make a minimal repro.
/cc @ocombe

**Comment by dslomov** [comment]
This should be fixed in bazel at HEAD.

**Comment by meteorcloudy** [comment]
Right, relevant change is https://github.com/bazelbuild/bazel/commit/c2c938ae2e75b5b881f06b18cce86dc87bae6fe6

**Comment by alexeagle** [comment]
Yes, works as an `sh_binary`, thanks!
